Tests/RevenueCatUITests/PaywallViewEventsTests.swift

```diff
@@ -27,7 +27,7 @@ class PaywallViewEventsTests: TestCase {
     private var handler: PurchaseHandler!
 
     private let mode: PaywallViewMode = .random
-    private let scheme: ColorScheme = Bool.random() ? .dark : .light
+    private let scheme: ColorScheme = .light
```

Comment:
Nit: instead of only running the test for the light ColorScheme, should we instead create two tests, one for each color scheme?

Reply:
ON IT